web_src/js/utils/image.js

```diff
@@ -0,0 +1,34 @@
+import pngChunksExtract from 'png-chunks-extract';
+
+export async function imageInfo(file) {
+  let width = 0;
+  let dppx = 1;
+
+  try {
+    if (file.type === 'image/png') {
```

Comment:
Instead of this nesting, what about `if (!file.type === 'image/png') return {width, dppx}`?
Then we remove a level of nesting.

Reply:
Agree. I prepared it for more types, but that can be done easily later.